newrelic_lambda_cli/integrations.py

```diff
@@ -531,7 +582,7 @@ def install_log_ingestion(
                     nr_license_key,
                 )
             except Exception as e:
-                failure("Failed to create 'newrelic-log-ingestion' function: %s" % e)
+                failure("Failed to create newrelic-log-ingestion function: %s" % e)
                 return False
         else:
```

Comment:
Let's change the failure message here to match the new behavior:
```suggestion
        else:
            failure(
                "CloudFormation Stack NewRelicLogIngestion exists (status: %s).\n"
                "Please manually delete the stack and re-run this command."
                % stack_status
            )
            return False
```